conf/machine/include/rpi-base.inc

```diff
@@ -31,6 +31,10 @@ RPI_KERNEL_DEVICETREE_OVERLAYS ?= " \
     overlays/i2c-gpio.dtbo \
     overlays/i2c-rtc.dtbo \
     overlays/imx219.dtbo \
+    overlays/imx290.dtbo \
```

Comment:
Does it work on all the kernel versions we currently have in the layer?

Reply:
It should, why wouldn't it?

Reply:
So, do they exist in all the kernel versions master currently provides?